gridt/src/app/app.component.ts

```diff
@@ -65,25 +65,19 @@ export class AppComponent implements OnInit, OnDestroy {
     );
   }
 
-  ngOnDestroy() {
-    this.cancelAllSubscriptions();
-  }
-
   private cancelAllSubscriptions() {
-    if (this.isLoggedInSubscription) {
-      this.isLoggedInSubscription.unsubscribe();
-    }
+    this.subscriptions.map( subscription => subscription.unsubscribe());
   }
 
   private checkAuthOnResume(state: AppState) {
     this.zone.run( () => {
       if (state.isActive) {
-        this.isLoggedInSubscription = this.auth.isLoggedIn$.subscribe(loggedIn => {
+        this.subscriptions.push(this.auth.isLoggedIn$.subscribe(loggedIn => {
```

Comment:
Because this takes place in the `app.component` there is no need to deal with the subscribe/unsubscribe logic.